specification/network/resource-manager/Microsoft.Network/stable/2018-12-01/examples/PeerExpressRouteCircuitConnectionGet.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "connectionName": "60aee347-e889-4a42-8c1b-0aae8b1e4013",
+    "resourceGroupName": "rg1",
+    "api-version": "2018-12-01",
+    "subscriptionId": "subid1",
+    "circuitName": "ExpressRouteARMCircuitA",
+    "peeringName": "AzurePrivatePeering"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "name": "60aee347-e889-4a42-8c1b-0aae8b1e4013",
+        "id": "/subscriptions/subid1/resourceGroups/rg1/providers/Microsoft.Network/expressRouteCircuits/ExpressRouteARMCircuitA/peerings/AzurePrivatePeering/peerConnections/60aee347-e889-4a42-8c1b-0aae8b1e4013",
+        "etag": "W/\"6ffbbb06-da20-44ca-a34f-280c4653b1e9\"",
+        "properties": {
+          "provisioningState": "Succeeded",
+          "expressRouteCircuitPeering": {
+            "id": "/subscriptions/subid1/resourceGroups/rg1/providers/Microsoft.Network/expressRouteCircuits/ExpressRouteARMCircuitA/peerings/AzurePrivatePeering"
+          },
+          "peerExpressRouteCircuitPeering": {
+            "id": "/subscriptions/subid1/resourceGroups/rg1/providers/Microsoft.Network/expressRouteCircuits/ExpressRouteARMCircuitB/peerings/AzurePrivatePeering"
+          },
+          "addressPrefix": "20.0.0.0/29",
+          "circuitConnectionStatus": "Connected",
+          "connectionName": "circuitConnectionWestusEastus",
+          "authResourceGuid": ""
```

Comment:
empty intentionally?

Reply:
Yes. string will be empty when no auth is used.